tools/src/main/resources/ui/js/raw-report.js

```diff
@@ -41,23 +45,41 @@ $(document).ready(function() {
         }
       },
       {
-        name: 'sqlDataFrameDuration',
-        data: 'sqlDataFrameDuration',
+        name: "sparkUser",
+        data: "user",
+      },
+      {
+        name: "startTime",
+        data: "startTime",
+        type: 'numeric',
         searchable: false,
         render: function (data, type, row) {
-          if (type === 'display' || type === 'filter') {
-            return formatDuration(data)
+          if (type === 'display') {
+            return formatTimeMillis(data)
           }
           return data;
         },
       },
       {
-        name: 'sqlDataframeTaskDuration',
-        data: 'sqlDataframeTaskDuration',
+        name: recommendGPUColName,
+        data: 'gpuCategory',
+        render: function (data, type, row) {
```

Comment:
again I think this should be removed and just use recommendation from the output of tool so everything matches. We can change the core logic in there if needed

Reply:
Done! category is determined from the application field `estimatedinfo.recommendation`